src/containers/JobSearch/index.tsx

```diff
@@ -1,11 +1,28 @@
-import { ReactElement } from "react";
-import { useSelector } from "react-redux";
+import React, { ReactElement, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 import { animated, useSpring } from "react-spring";
+import { fetchJobs } from "../redux/actions/jobActions";
 import { RootStore } from "../redux/store";
 import "./jobSearch.scss";
 
 const JobSearch: React.FC = () => {
+  //Get Data Jobs
+  const jobs = useSelector((state: RootStore) => state.allJobs);
+
+  const dispatch = useDispatch();
+  //Initialize the variable to receive text and set text
+  const [searchJobByTitle, setSearchJobByTitle] = useState<string>("");
+
+  //Initialize function to receive text on change
+  const handleChangeWithDebounce = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const valueTarget: string = e.target.value;
+    return setSearchJobByTitle(valueTarget);
```

Comment:
- Please change the name of the function. It is not debouncing.
- Why did you return on this?